src/specFunctions.py

```diff
@@ -208,11 +208,17 @@ def peakIndexes(y, thres=0.3, min_dist=1, thres_abs=False):
 
         # for each chain of zero indexes
         for plateau in zero_plateaus:
-            median = np.median(plateau)
-            # set leftmost values to leftmost non zero values
-            dy[plateau[plateau < median]] = dy[plateau[0] - 1]
-            # set rightmost and middle values to rightmost non zero values
-            dy[plateau[plateau >= median]] = dy[plateau[-1] + 1]
+            lplat = plateau.shape[0]
```

Comment:
I noticed that the data in `plateau` is always in sorted order, so we can skip finding medians and know that the middle element *is* the median.